static/js/components/analytics/AnalyticsChart.js

```diff
@@ -61,16 +61,13 @@ export class AnalyticsChart extends React.Component {
 
   render() {
     const { dimensions } = this.state
-    const {
-      analyticsData,
-      getColorForChannel,
-      duration,
-      ...remainingProps
-    } = this.props
-    const passThroughProps = _.omit(remainingProps, [
+    const passThroughProps = _.omit(this.props, [
```

Comment:
Are the changes to `AnalyticsChart.js` from another PR?

Reply:
@mbertrand If I recall correctly, I think I changed `AnalyticsChart` because there was a missing null-check that was causing tests that do full mount rendering to fail. So I think these changes should stick w/ this PR.

Does that make sense? Let me know what you think and I'll handle accordingly.

Reply:
Sounds good and analytics chart still works, approving.